_includes/pbc/all/install-features/202304.0/install-the-order-management-and-inventory-management-feature.md

```diff
@@ -0,0 +1,70 @@
+
+
+
+This document describes how to ingrate the [Order Management](/docs/scos/user/features/{{page.version}}/order-management-feature-overview/order-management-feature-overview.html) + [Inventory Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html) features into a Spryker project.
+
+{% info_block errorBox %}
+
+The following features integration guide expects the basic feature to be in place.
+
+The current feature integration guide adds the following functionality:
+
+* [Warehouse Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html)
```

Comment:
the intro sentence says we integrate Order Management + Inventory Management, whereas here, it says Warehouse Management and Order Management. Which one is correct?

Reply:
Inventory Management is the right one. Fixed